circuits/cpp/barretenberg/cpp/bootstrap.sh

```diff
@@ -1,6 +1,25 @@
 #!/bin/bash
 set -eu
 
+# Get the clang version string
+clang_version_string=$(clang --version 2>/dev/null)
```

Comment:
nitpick: technically, since this script has -e, we would never hit the 
```

# Check if clang is installed
if [ $? -ne 0 ]; then
  echo "Error: clang is not installed."
  exit 1
fi
```
case

Reply:
I'll add the set -e after this